models/tenant.py

```diff
@@ -36,8 +34,6 @@ def json(self):
             'phone': self.phone,
             'propertyID': self.propertyID,
             'propertyName': self.property.name if self.property else None,
```

Comment:
Do we want to get rid of this `propertyName` too?

Reply:
Page 17 of the Figma docs suggests we should keep this on the Tenant model, but I could be thinking about this wrong.

I believe the idea of accomplishing page 17 would be to GET Users/?id and the returned data would have a tenants property that lists all of the data needed in the `Tenants` section of page 17, including propertyName.

Let me know if you see a better way of accomplishing this, and I'd be happy to re-work this as needed!